invenio_github/errors.py

```diff
@@ -32,7 +32,7 @@ class GitHubError(Exception):
 class RepositoryAccessError(GitHubError):
     """Repository access permissions error."""
 
-    message = u'The user cannot access the github repository'
+    message = "The user cannot access the github repository"
```

Comment:
This string does not include any of the attributes of the error class. Are those included somehow by the parent? (Happens also to other errors in this file)

Reply:
Here I was not sure whether to include specific info or not. E.g. some errors might be revealed to the user, thus exposing things like `user_id` or `repository_id`. I do agree that, if we are not using that information, it'd be better if we remove it.  

Since it was already like this, I just followed what we had and postpone that decision for later.

Reply:
Created an [issue](https://github.com/inveniosoftware/invenio-github/issues/106)